app/components/message_attachments/action_button/action_button.js

```diff
@@ -20,6 +21,7 @@ export default class ActionButton extends PureComponent {
         theme: PropTypes.object.isRequired,
         cookie: PropTypes.string.isRequired,
         disabled: PropTypes.bool,
+        style: PropTypes.string,
```

Comment:
`style` is a very overloaded word, and tends to be reserved for theming/styling of entire components as a whole, rather than just a singular colour styling element, as is the case here.

Could we rename to `actionColor`, `actionButtonColor` or something similar? Maybe even just `color` is appropriate.

Reply:
Thank you! Renamed to `buttonColor`.